test/dotcom/system_status_test.exs

```diff
@@ -0,0 +1,90 @@
+defmodule Dotcom.SystemStatusTest do
+  use ExUnit.Case
+
+  import Dotcom.Alerts, only: [service_impacting_effects: 0]
+  import Dotcom.SystemStatus
+
+  import Dotcom.Utils.ServiceDateTime,
+    only: [
+      service_range_day: 0,
+      service_range_next_week: 0
+    ]
+
+  import Mox
+  import Test.Support.Factories.Alerts.Alert
+
+  setup :verify_on_exit!
+
+  describe "subway_alerts_for_today/0" do
+    test "requests alerts for all subway lines @ today datetime" do
+      today = Faker.DateTime.forward(1)
+
+      expect(Dotcom.Utils.DateTime.Mock, :now, fn ->
+        today
+      end)
+
+      expect(Alerts.Repo.Mock, :by_route_ids, fn route_ids, ^today ->
+        assert Enum.sort(route_ids) == Dotcom.Routes.subway_route_ids() |> Enum.sort()
+        []
+      end)
+
+      _ = subway_alerts_for_today()
+    end
+
+    # does Alerts.Repo.by_route_ids/2 not already scope results to the given day?
+    test "filters alerts for falling within a day" do
+      stub_with(Dotcom.Utils.DateTime.Mock, Dotcom.Utils.DateTime)
```

Comment:
You can move these up into a `setup` block and they will be overridden only if you do an `expect` in the test.